neural_compressor/adaptor/pytorch.py

```diff
@@ -4559,21 +4560,44 @@ def rtn_quantize(self, model, tune_cfg):
         return model
 
     def gptq_quantize(self, model, tune_cfg, dataloader):
+        import pdb;pdb.set_trace()
         logger.debug("quantizing with the GPTQ algorithm")
         from .torch_utils.weight_only import gptq_quantize
-        if 'gptq_args' in self.recipes:
-            percdamp = self.recipes['gptq_args'].get('percdamp', 0.01)
-            wbits = self.recipes.get('wbits', 4)
-            group_size = self.recipes.get('group_size', 128)
-            sym = self.recipes.get('scheme', False)
-        # implementation of gptq
-        # GPTQ(model, dataloader, w_bit, group_size, percdamp=0.01)
+        # convert tune_cfg to gptq_quantize's weight config
+        """please refer to weight_config which can be analyzed by user-define API function weight_only.gptq_quantize
+        keys of weight_config can not only be specific name, but can also be a re formula
         weight_config = {
-            'wbits': wbits, 
-            'group_size': group_size, 
-            'sym': sym,
-            'percdamp': percdamp
+            "layer_name_1": {
+                'wbits': 4,
+                'group_size': 128,
+                'sym': False,
+                'percdamp': 0.01,
+                'actorder': True
+            },
+            "layer_name_2": {
+                'wbits': 4,
+                'group_size': 128,
+                'sym': False,
+                'percdamp': 0.01,
```

Comment:
what does it mean?

Reply:
I guess it is to  improve the robustness of the inversing of matrixes by adding this value to the diag